client/components/organization-wrapper/organization-wrapper.js

```diff
@@ -76,8 +77,12 @@ export default class OrganizationWrapper extends React.Component {
                     } if (isAuthenticated && needsMobilePhoneVerification) {
                         return <Redirect to={`/${orgSlug}/mobile-phone-verification`} />;
                     }
-                    return <Registration {...props} />;
-                  }}
+                    return (<React.Suspense fallback={<Loader full={false} />}>
+                              <Registration {...props} />
+                            </React.Suspense>
+                          );
```

Comment:
please let's use an indentation which is consistent with the rest of the codebase, eg:

```javascript
return (
  <React.Suspense fallback={<Loader full={false} />}>
    <Registration {...props} />
  </React.Suspense>
);
```

Applies to all other occurrences of `React.Suspense`.